src/main/java/com/mraof/minestuck/tileentity/CassettePlayerTileEntity.java

```diff
@@ -0,0 +1,44 @@
+package com.mraof.minestuck.tileentity;
+
+import net.minecraft.inventory.IClearable;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.tileentity.TileEntity;
+
+public class CassettePlayerTileEntity extends TileEntity implements IClearable {
```

Comment:
Class needs reformatting

Reply:
does it still need it? I reformatted it on my last upload

Reply:
Oh, forgot to resolve this comment